File: 1019-squares-of-a-sorted-array/squares-of-a-sorted-array.java
class Solution {
    public int[] sortedSquares(int[] nums) {
        int t,i,j=nums.length;
        for(i=0;i<nums.length;i++){
            nums[i]=nums[i]*nums[i];
        }
        Arrays.sort(nums);
        return nums;
    }
}

File: 1019-squares-of-a-sorted-array/squares-of-a-sorted-array.cpp
class Solution {
public:
    vector<int> sortedSquares(vector<int>& a) {
        
        int i=0,j,k,c=0,l,h,m;
        int n=a.size();
        vector<int> v(n);
        

        j=n-1;

        while(i<=j)
        {
            if(a[i]<0)
            a[i]*=(-1);
            if(a[j]<0)
            a[j]*=(-1);

            if(a[i]<a[j])
            {
                v[c++]=a[j]*a[j];
                j--;
            }
            else
            {
                v[c++]=a[i]*a[i];
                i++;
            }
        }
        c=0;
        for(i=n-1;i>=0;i--)
        a[c++]=v[i];
        return a;
    }
};